Dockerfile

```diff
@@ -38,18 +40,22 @@ EXPOSE 3001
 EXPOSE 3002
 EXPOSE 3005
 WORKDIR /opt/one-app
+USER $USER
 RUN chown node:node /opt/one-app
 CMD ["node", "lib/server"]
 COPY --from=builder --chown=node:node /opt/one-app/development ./
 
 # production image
 # last so that it's the default image artifact
 FROM node:12-alpine as production
+ARG USER
+ENV USER ${USER:-node}
```

Comment:
`-node` is this the default fallback arg?

Reply:
yes it is, if no value is provided it would default to it